### PR TITLE
Clean up and document populate-uuids.js

### PR DESCRIPTION
**Pull request by ripleyaffect**
No functional changes

### CODE REVIEW

File: src/populate-uuids.js [hunk 1]
```diff
@@ -1,115 +1,162 @@
...
+ */
+const createUUIDs = ($, options) => {
+  // Get the first node with no UUID
+  let node = getNextNode($);
+
+  // Find all uuids that exist already in the DOM
+  const existingUUIDs = _.map($('[uuid]'), el => el.getAttribute('uuid'));
+
+  // Initialize the UUID varialbe
+  let uid;
```
Comment (mikebranski) [inline src/populate-uuids.js]: Is this intentionally `uid` instead of `uuid`?

File: src/populate-uuids.js [hunk 2]
```diff
@@ -1,115 +1,162 @@
...
     try {
-      xmlStr = populate(originalXml, options);
+      newXml = populate(originalXml, options);
     } catch (e) {
-      gutil.log(gutil.colors.red('Strict UUID Check Failed: ' + e.message));
-      return done(
-        new PluginError('gulp-populate-uuids', 'Strict UUID Check Failed.')
-      );
+      const errorMessage = 'Strict UUID Check Failed';
+      log(colors.red(`${errorMessage}: ${e.message}`));
```
Comment (mikebranski) [inline src/populate-uuids.js]: Could use [`log.error()`](https://www.npmjs.com/package/fancy-log#logerrormsg) instead of `log()`.

Comment (ripleyaffect) [reply src/populate-uuids.js]: yup, I'm deprecating `gulp-util`, and replacing with the individual libs that are recommended, but will be in a separate pr.

File: src/populate-uuids.js (hunk 1 above)
Comment (ripleyaffect) [reply src/populate-uuids.js]: yeah. This was because `uuid` was used as the name for an import. But now it's `UUID`, so I guess I could change this

**Pull request merged**